puppet/zulip/manifests/tornado_sharding.pp

```diff
@@ -11,7 +11,7 @@
     ensure  => file,
     owner   => 'root',
     group   => 'root',
-    mode    => '0644',
+    mode    => '0640',
```

Comment:
Hmm, I would think we wanted the 0644 version so that the `zulip` user can read this file?

Reply:
Yeah, reasonable.  Why is `/etc/zulip/sharding.json` 640?

Reply:
@mateuszmandera may remember, but I think it's probably a mistake and both of them should be 644.  The pattern in /etc/zulip is that the secrets file is 640 and everything else is 644.

Reply:
https://chat.zulip.org/#narrow/stream/3-backend/topic/focal.20support/near/901797 for context. Both were `640` originally, but nginx_sharding.conf was changed to 644 because it needs to be readable by the `zulip` user as Tim said. `sharding.json` is already owned by `zulip`, so the adjustment wasn't necessary. Makes sense to have `644` for consistency and to avoid having to puzzle over such questions again I think.